internal/proto/local/controller/api/resources/sessions/v1/session.proto

```diff
@@ -23,6 +23,35 @@ message SessionState {
   google.protobuf.Timestamp end_time = 30 [json_name = "end_time"];
 }
 
```

Comment:
Mind adding a doc comment for the `Connection` message?

Reply:
Done.